Practice/case.py

```diff
@@ -55,32 +77,74 @@ def make_list(case, word):
 
 
 
-def output_formatted_words(word_list):
+def output_formatted_word(snake, camel, kebob, constant):
+
+    print('Snake: ' + snake)
+    print('Camel: ' + camel)
+    print('Kebob: ' + kebob)
+    print('Constant' + constant)
+
+def set_snake(word_list):
     """
+    >>> set_snake(['this', 'is', 'a', 'test'])
+    'this_is_a_test'
 
-    >>> output_formatted_words(['this', 'is', 'a', 'test'])
-    snake: this_is_a_test
-    kebob: this-is-a-test
-    camel: ThisIsATest
-    constant: THIS_IS_A_TEST
+    :param word_list:
+    :return: snake as a string
+    """
+    snake = ('snake: ' + '_'.join(word_list))
+    return snake
+
+
+def set_kebob(word_list):
+    """
+    >>> set_kebob(['this', 'is', 'a', 'test'])
+    'this-is-a-test'
 
     :param word_list:
-    :return:
+    :return: kebob as a string
+    """
+    kebob = 'kebob: ' + '-'.join(word_list)
+    return kebob
+
+
+def set_camel(word_list):
     """
-    print('snake: ' + '_'.join(word_list))
-    print('kebob: ' + '-'.join(word_list))
+       >>> camel(['this', 'is', 'a', 'test'])
+       'ThisIsATest'
+
+       :param word_list:
+       :return: camel as a string
+       """
     word_list = [element.title() for element in word_list]
-    print('camel: ' + ''.join(word_list))
-    print('constant: ' + ('_'.join(word_list)).upper())
+    camel = 'camel: ' + ''.join(word_list)
+    return camel
+
+
+def set_constant(word_list):
```

Comment:
These `set_*` functions could have a better name. "Set" implies assign or fix. You're doing more like "converting" or "rendering" of words.